src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -274,15 +276,18 @@
                                          :title "Synced"})]
                  :else [:> Sync (merge (use-style sync-icon-style)
                                        {:style {:color (color :highlight-color)}
-                                        :title "Synchronizing..."})])]]]
+                                        :title "Synchronizing..."})])]]]]
+
 
-            [separator]]
            [button {:style {:min-width "max-content"} :on-click #(dispatch [:get-db/init]) :primary true} "Load Test DB"])
          [button {:on-click #(dispatch [:theme/toggle])
                   :title "Toggle Color Scheme"}
           (if @theme-dark
             [:> ToggleOff]
             [:> ToggleOn])]
+         [:> Tooltip {:title "Help", :arrow true}
+          [:span [button {:on-click #(dispatch [:help/toggle])}
+                  [:> Help]]]]
```

Comment:
Lets use the `title` field for tooltips until/unless we decide to change all of them at once.